# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acroread/mime-database.diff
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Adobe/Reader8/Reader/intellinux/bin/xdg-mime.orig	2008-06-25 18:17:05.000000000 +0200
+++ Adobe/Reader8/Reader/intellinux/bin/xdg-mime	2008-09-02 21:56:55.000000000 +0200
@@ -1084,7 +1084,7 @@
         ;;
 esac
 
-update_mime_database $xdg_base_dir
+echo update_mime_database $xdg_base_dir
 
 exit_success
 
